core/res/null.json

```diff
@@ -19,7 +19,7 @@
       "generic": "0x0"
     },
     "score": "0x20000",
-    "author": "0x0000000000000000000000000000000000000000",
+    "author": "cccqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6n0r0d",
```

Comment:
Why do you use `ccc` prefix here? It should start with `ccc` to be consistent with other files.